src/Kernel32.Desktop/Kernel32+JobObjectInfoClass.cs

```diff
@@ -0,0 +1,87 @@
+
+            /// <summary>
+            /// This flag is not supported. Applications must set security limitations individually for each process.
+            /// </summary>
+            JobObjectSecurityLimitInformation = 5,
+
+            /// <summary>
+            /// The lpJobObjectInfo parameter is a pointer to a JOBOBJECT_END_OF_JOB_TIME_INFORMATION structure.
+            /// </summary>
+            JobObjectEndOfJobTimeInformation = 6,
+
+            /// <summary>
+            /// The lpJobObjectInfo parameter is a pointer to a JOBOBJECT_ASSOCIATE_COMPLETION_PORT structure.
+            /// </summary>
+            JobObjectAssociateCompletionPortInformation = 7,
+
+            /// <summary>
+            /// The lpJobObjectInfo parameter is a pointer to a JOBOBJECT_EXTENDED_LIMIT_INFORMATION structure.
+            /// </summary>
+            JobObjectExtendedLimitInformation = 9,
+
+            /// <summary>
+            /// The lpJobObjectInfo parameter is a pointer to a USHORT value that specifies the list of processor groups to assign the job to.
+            /// The cbJobObjectInfoLength parameter is set to the size of the group data. Divide this value by sizeof(USHORT) to determine the number of groups.
+            /// </summary>
+            JobObjectGroupInformation = 11,
+
+            /// <summary>
+            /// The lpJobObjectInfo parameter is a pointer to a JOBOBJECT_NOTIFICATION_LIMIT_INFORMATION structure.
+            /// </summary>
+            JobObjectNotificationLimitInformation = 12,
+
+            /// <summary>
+            /// The lpJobObjectInfo parameter is a pointer to a buffer that contains an array of GROUP_AFFINITY structures that specify the affinity of the job for the processor groups to which the job is currently assigned.
+            /// The cbJobObjectInfoLength parameter is set to the size of the group affinity data. Divide this value by sizeof(GROUP_AFFINITY) to determine the number of groups.
+            /// </summary>
+            JobObjectGroupInformationEx = 14,
+
+            /// <summary>
+            /// The lpJobObjectInfo parameter is a pointer to a JOBOBJECT_CPU_RATE_CONTROL_INFORMATION structure.
+            /// </summary>
+            JobObjectCpuRateControlInformation = 15,
+
+            /// <summary>
+            /// The lpJobObjectInfo parameter is a pointer to a JOBOBJECT_NET_RATE_CONTROL_INFORMATION structure.
+            /// </summary>
+            JobObjectNetRateControlInformation = 32,
+
+            /// <summary>
+            /// The lpJobObjectInfo parameter is a pointer to a JOBOBJECT_NOTIFICATION_LIMIT_INFORMATION_2 structure.
+            /// </summary>
+            JobObjectNotificationLimitInformation2 = 34,
+
+            /// <summary>
+            /// The lpJobObjectInfo parameter is a pointer to a JOBOBJECT_LIMIT_VIOLATION_INFORMATION_2 structure.
+            /// </summary>
+            ViolationInJobObjectLimitViolationInformation2formation2 = 35
```

Comment:
Actually these are the names from MSDN :

https://msdn.microsoft.com/en-us/library/windows/desktop/ms686216(v=vs.85).aspx